appveyor.yml

```diff
@@ -1,64 +0,0 @@
-
```

Comment:
That's weird. Actually I don't lose AppVeyor at all when re-rendering and it appears we do have `python` in `build` already. 😕